spec/automated_review/id_columns_are_bigint_spec.rb

```diff
@@ -1,39 +1,58 @@
 require 'set'
+
 describe "*_id columns" do
-  EXCEPTIONS_LIST = Set.new(
-    [
-      "cloud_networks.provider_segmentation_id",
-      "container_volumes.common_volume_id",
-      "miq_queue.task_id",
-      "network_ports.binding_host_id",
-      "scan_histories.task_id",
-      "sessions.session_id"
-    ]
-  ).freeze
+  let(:exceptions_list) do
+    Set.new(%w[
+      cloud_networks.provider_segmentation_id
+      container_volumes.common_volume_id
+      miq_queue.task_id
+      network_ports.binding_host_id
+      scan_histories.task_id
+      sessions.session_id
+    ])
+  end
 
   it "should be type bigint" do
+    not_bigint = []
+    no_longer_exceptions = []
```

Comment:
Added this so that we can collect all of the "bad" ones and present them all at once as opposed to failing for each individual one.  Once I verified this test worked, I coped this test over to the new fk test.